src/controllers/restful/PaymentController.ts

```diff
@@ -0,0 +1,204 @@
+    errorOrPaymentCheckPagoPaResponse.value,
+    errorOrCodiceContestoPagamento.value
+  );
+  if (errorOrPaymentCheckResponse.isLeft()) {
+    return left(
+      RestfulUtils.sendErrorResponse(
+        res,
+        ControllerError.ERROR_INVALID_API_RESPONSE,
+        HttpErrorStatusCode.INTERNAL_ERROR
+      )
+    );
+  }
+  RestfulUtils.sendSuccessResponse(res, errorOrPaymentCheckResponse.value);
+  return right(errorOrPaymentCheckResponse.value);
+}
+
+// Forward a payment check request from BackendApp to PagoPa
+export async function activatePaymentToPagoPa(
+  req: express.Request,
+  res: express.Response,
+  pagoPaConfig: PagoPaConfig
+): Promise<Either<ControllerError, PaymentsActivationResponse>> {
+  // Validate input
+  const errorOrPaymentsActivationRequest = PaymentsActivationRequest.decode(
+    req.params
+  );
+  if (errorOrPaymentsActivationRequest.isLeft()) {
+    return left(
+      RestfulUtils.sendErrorResponse(
+        res,
+        ControllerError.ERROR_INVALID_INPUT,
+        HttpErrorStatusCode.BAD_REQUEST
+      )
+    );
+  }
+
+  // Convert controller request to PagoPa request
+  const errorOrPaymentsActivationRequestPagoPa = PaymentsConverter.getPaymentsActivationRequestPagoPa(
+    pagoPaConfig,
+    errorOrPaymentsActivationRequest.value
+  );
+  if (errorOrPaymentsActivationRequestPagoPa.isLeft()) {
+    return left(
+      RestfulUtils.sendErrorResponse(
+        res,
+        ControllerError.ERROR_INVALID_INPUT,
+        HttpErrorStatusCode.BAD_REQUEST
+      )
+    );
+  }
+
+  // Require payment activation to PagoPa API
+  const errorOrPaymentActivationPagoPaResponse = await PaymentsService.sendPaymentsActivationRequestToPagoPaAPI(
+    errorOrPaymentsActivationRequestPagoPa.value,
+    pagoPaConfig
+  );
+
+  // Provide a response to applicant
+  if (errorOrPaymentActivationPagoPaResponse.isLeft()) {
+    RestfulUtils.sendUnavailableAPIError(res);
+    return left(ControllerError.ERROR_API_UNAVAILABLE);
+  }
+
+  // Check if request was rejected
+  if (
+    errorOrPaymentActivationPagoPaResponse.value.nodoAttivaRPTRisposta.esito ===
+    PPTPortTypes.Esito.KO
+  ) {
+    return left(
+      RestfulUtils.sendErrorResponse(
+        res,
+        ControllerError.REQUEST_REJECTED,
+        HttpErrorStatusCode.BAD_REQUEST
+      )
+    );
+  }
+
+  // Convert PagoPa response to controller response
+  const errorOrPaymentActivationResponse = PaymentsConverter.getPaymentsActivationResponse(
+    errorOrPaymentActivationPagoPaResponse.value
+  );
+  if (errorOrPaymentActivationResponse.isLeft()) {
+    return left(
+      RestfulUtils.sendErrorResponse(
+        res,
+        ControllerError.ERROR_INVALID_API_RESPONSE,
+        HttpErrorStatusCode.INTERNAL_ERROR
+      )
+    );
+  }
+  RestfulUtils.sendSuccessResponse(res, errorOrPaymentActivationResponse.value);
+  return right(errorOrPaymentActivationResponse.value);
+}
+
+/** Receive an async activation result frop PagoPA
```

Comment:
typo, also bad formatting of multiline jsdoc commen, see [here](http://usejsdoc.org/howto-es2015-classes.html) for examples